src/components/HighlightedName/index.tsx

```diff
@@ -0,0 +1,71 @@
+import ColorHash from 'color-hash';
+import assign from 'lodash/assign';
+import memoize from 'lodash/memoize';
+import * as React from 'react';
+import styled from 'styled-components';
+import asRendition from '../../asRendition';
+import { Coloring, RenditionSystemProps, Theme } from '../../common-types';
+import { getColor, isLight } from '../../utils';
+import Txt, { TxtProps } from '../Txt';
+
+const colorHash = new ColorHash();
+const backgroundColor = memoize((text: string): string => {
+	return colorHash.hex(text.replace(/\s/g, ''));
+});
+
+const BaseHighlightedName = styled(Txt.span)`
+	font-family: ${props => props.theme.titleFont};
+	display: inline-block;
+	border-radius: 2px;
+	line-height: 1;
+	color: ${props => props.color};
```

Comment:
I don't think this is necessary, as passing `color` to the `Txt` component should be enough

Reply:
When `isLight` returns true, it returns `#fff`, but the `#fff` as the `color` prop. But the color prop never gets fed to the styled component. I think this is broken in the old Badge component.